gulpfile.js

```diff
@@ -364,8 +364,8 @@ gulp.task('build', gulp.series(
     'build:hexo',
     'copy:robots.txt',
     // 'generate-service-worker',
-    'compress:zopfli',
-    'compress:brotli',
+    // 'compress:zopfli',
```

Comment:
I think you forgot to uncomment these lines

Reply:
🤦‍♂️